Packs/CarbonBlackDefense/Integrations/CarbonBlackDefenseV2/CarbonBlackDefenseV2.py

```diff
@@ -0,0 +1,1169 @@
+                                      human_readable,
+                                      headers=headers,
+                                      removeNull=True,
+                                      headerTransform=string_to_table_header)
+
+    return CommandResults(
+        outputs_prefix='CarbonBlackDefense.Process.Results',
+        outputs_key_field='id',
+        outputs=res,
+        readable_output=readable_output,
+        raw_response=res
+    )
+
+
+def alerts_search_command(client: Client, args: dict):
+    alert_type = args.get('type', 'all')
+    category = argToList(args.get('category'))
+    device_id = argToList(args.get('device_id'))
+    first_event_time = args.get('first_event_time')
+    policy_id = argToList(args.get('policy_id'))
+    process_sha256 = argToList(args.get('process_sha256'))
+    reputation = argToList(args.get('reputation'))
+    tag = argToList(args.get('tag'))
+    device_username = argToList(args.get('device_username'))
+    query = args.get('query')
+    rows = args.get('rows')
+    start = args.get('start')
+    headers = ['id', 'category', 'device_id', 'device_name', 'device_username', 'create_time', 'ioc_hit', 'policy_name',
+               'process_name', 'type', 'severity']
+    human_readable = []
+
+    if first_event_time:
+        first_event_time = json.loads(first_event_time)
+    res = client.get_alerts(alert_type, category, device_id, first_event_time, policy_id, process_sha256, reputation,
+                            tag, device_username, query, rows, start)
+
+    alerts = res.get('results', [])
+    if not alerts:
+        return 'No alerts were found.'
+    for alert in alerts:
+        human_readable.append({
+            'id': alert.get('id'),
+            'category': alert.get('category'),
+            'device_id': alert.get('device_id'),
+            'device_name': alert.get('device_name'),
+            'device_username': alert.get('device_username'),
+            'create_time': alert.get('create_time'),
+            'ioc_hit': alert.get('ioc_hit'),
+            'policy_name': alert.get('policy_name'),
+            'process_name': alert.get('process_name'),
+            'type': alert.get('type'),
+            'severity': alert.get('severity')
+        })
+
+    readable_output = tableToMarkdown('Carbon Black Defense Alerts List Results', human_readable, headers,
+                                      headerTransform=string_to_table_header, removeNull=True)
+    return CommandResults(
+        outputs_prefix='CarbonBlackDefense.Alert',
+        outputs_key_field='id',
+        outputs=alerts,
+        readable_output=readable_output,
+        raw_response=res
+    )
+
+
+def get_alert_details_command(client: Client, args: dict):
+    alert_id = args.get('alertId')
+
+    if not alert_id:
+        return "Missing required arguments."
+    res = client.get_alert_by_id(alert_id)
+
+    if 'id' not in res.keys():
+        return 'The alert you requested was not found'
```

Comment:
Use fstring here and add the alert id:
```suggestion
        return (f'The alert id: {alert_id} was not found')
``